perun-core/src/test/java/cz/metacentrum/perun/core/entry/GroupsManagerEntryIntegrationTest.java

```diff
@@ -2088,6 +2074,152 @@ public void getAssignedGroupsToFacility() throws Exception {
 
 	}
 
+	@Test (expected = GroupMoveNotAllowedException.class)
+	public void testMoveGroupTodiffVo() throws Exception {
+		System.out.println("\ntest moveGroup() to different vo");
```

Comment:
We usually print test name like this: ``System.out.println(CLASS_NAME + "testMoveGroupTodiffVo");`` Please fix it in a rest of tests.